fastlane/Fastfile

```diff
@@ -0,0 +1,133 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:ios)
+
+platform :ios do
+
+  before_all do
+    skip_docs
+    import_shared_fastfile
+  end
+
+  ######################### PUBLIC LANES ##########################
+
+  desc "Release a new version of the ios-sdk. Uploading a new release to a GitHub and CocoaPods trunk"
+  desc "Before doing so don't forget to follow check all steps in RELEASING.md"
+  lane :release do
+    ensure_branch
+
+    cocoapods_lint
+
+    # Move upcoming to releases in changelog
+    upcoming_version = get_upcoming_version.clone
+    put_upcoming_to_releases
+
+    podspecs_version_bump(version: upcoming_version)
+
+    release_branch = ENV['GITHUB_RELEASE_BRANCH']
+    remote = ENV['GITHUB_REMOTE']
+    git_add_or_update_tag(
+      tag_name: upcoming_version,
+      remote: remote,
+      branch: release_branch
+    )
+
+    push_to_git_remote(
+      remote: remote,
+      local_branch: release_branch,
+      remote_branch: release_branch
+    )
+
+    make_github_release(version: upcoming_version)
+
+    podspecs_push
+  end
+
+  ######################### PRIVATE LANES ##########################
+
+  desc "Build frameworks with Carthage"
+  private_lane :build_frameworks do
+    framework_name = "#{ENV['ITININERARIES_FRAMEWORK']}.framework.zip"
```

Comment:
I think there's a typo `ITININERARIES_FRAMEWORK`
Should be `ITINERARIES_FRAMEWORK`

Reply:
Nice catch. Thanks and updated in 1e4ec1c.